frontend/src/lib/utils.ts

```diff
@@ -36,7 +36,7 @@ export const mediaIdToUrlConverter = (
 }
 
 export const removeDuplicates = (arr: Media[]) => {
-  let unique_ids = []
+  const unique_ids = []
```

Comment:
U crazy, CodeFactor

Reply:
I hope this still works as intended... Gotta admit I don't know the consequence of this is 🙈

Reply:
Should we just yolo it? :D

Reply:
Yea go for it :D

Reply:
🙏

Reply:
Sweet 🙌